guidelines/sc/22/findable-help.html

```diff
@@ -0,0 +1,20 @@
+<section class="sc changed">
+   
+    <h4>Findable Help</h4>
+    
+    <p class="conformance-level">A</p>
+	<p class="change">New</p>
+    
+    <p>For <a href="https://www.w3.org/TR/WCAG21/#single-page-apps">single page apps</a> or any <a href="https://www.w3.org/TR/WCAG21/#dfn-set-of-web-pages">set of web pages</a> with blocks of content that are repeated on multiple web pages, if one of the following is available, 
```

Comment:
Are those links supposed to go to WCAG 2.1, considering this is a Success Criterion that is supposed to be published as part of WCAG 2.2?

Reply:
The links get re-addressed as part of the build process. I'll nullify them for now to avoid confusion.